src/libraries/System.Drawing.Common/tests/FontTests.cs

```diff
@@ -978,5 +978,18 @@ private static void VerifyFont(Font font, string expectedName, float expectedEmS
             Assert.False(font.IsSystemFont);
             Assert.Empty(font.SystemFontName);
         }
+
+        [ConditionalFact(Helpers.IsDrawingSupported)]
+        public void GetHashCode_DifferentNameSameSizeStyleUnit_HashCodeIsNotSame()
```

Comment:
It seems like you need to add
```cs
[SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "GetHashCode doesn't include font name in .NET Framework")]
```

Reply:
Oh, that's super helpful @safern. I'll add that.